CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ This changelog adheres to [Keep a CHANGELOG](http://keepachangelog.com/).
 
 ## [Unreleased]
 
+## [0.13.0] - released on 2017-07-12
+### Added
+ - A metadata file is generated when producing artifacts which contains metadata about components and various other information about a built artifact.
```

Comment:
The rule I usually used was "if there's no documentation written for it in the README, document it in the changelog"